tests/ui/blocks_in_if_conditions.rs

```diff
@@ -63,10 +63,10 @@ fn block_in_assert() {
     let opt = Some(42);
     assert!(opt
         .as_ref()
-        .and_then(|val| {
+        .map(|val| {
```

Comment:
Was it changed because it triggered the `bind_instead_of_map` lint?

Reply:
Yeap